### PR TITLE
[ADD] module 'sale_order_amount_uos_based'

### PR DESCRIPTION
**Pull request by tafaRU**

### CODE REVIEW

File: sale_order_amount_uos_based/model/sale_order_line.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+class SaleOrderLine(orm.Model):
+    _inherit = 'sale.order.line'
+
+    def _amount_line(self, cr, uid, ids, field_name, arg, context=None):
+        uom_qty_dict = {}
+        if context is None:
+            context = {}
+        for line in self.browse(cr, uid, ids, context=context):
+            uom_qty_dict[line.id] = line.product_uom_qty
+            cr.execute(
```
Comment (eLBati) [inline sale_order_amount_uos_based/model/sale_order_line.py]: Please add a comment explaining what is the purpose of this

File: sale_order_amount_uos_based/model/sale_order_line.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+##############################################################################
+
+from openerp.osv import orm, fields
+import openerp.addons.decimal_precision as dp
+
+
+class SaleOrderLine(orm.Model):
+    _inherit = 'sale.order.line'
+
+    def _amount_line(self, cr, uid, ids, field_name, arg, context=None):
```
Comment (eLBati) [inline sale_order_amount_uos_based/model/sale_order_line.py:28]: I would enable this behaviour if the user belongs to 'Manage Secondary Unit of Measure'

File: sale_order_amount_uos_based/README.rst [hunk 3]
```diff
@@ -0,0 +1,35 @@
+Module name
```
Comment (pedrobaeza) [inline sale_order_amount_uos_based/README.rst]: You have to put here the module name or a module title.

File: sale_order_amount_uos_based/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+#
+##############################################################################
+{
+    'name': "Sale Order Amount Uos Based",
+    'version': '1.0',
+    'category': 'Sales Management',
+    'author': 'Agile Business Group',
+    'website': 'http://www.agilebg.com',
+    'license': 'AGPL-3',
+    "depends": [
```
Comment (pedrobaeza) [inline sale_order_amount_uos_based/__openerp__.py:31]: This is 7.0, so you need to put also description key here.

**Comment by pedrobaeza** [comment]
I find your implementation a little ugly and hackish (with an SQL update and so on). Please see our module for v8 with an alternative approach for UoS: https://github.com/odoomrp/odoomrp-wip/tree/8.0/sale_uos_price

**Comment by eLBati** [comment]
I think it's better to put this in a custom module.
@tafaRU if you agree, please close PR

**Pull request closed without merging**